### PR TITLE
Test relay parachain

### PR DESCRIPTION
**Pull request by girazoki**
Adds one parachain to produce blocks with appropriate collators (at least until we can assign then from tanssi)

Detected an error based of the order of the workers

### CODE REVIEW

File: test/configs/zombieStarlight.json [hunk 1]
```diff
@@ -46,15 +46,45 @@
...
         {
-            "id": 2000,
+            "id": 1000,
             "chain_spec_path": "specs/single-container-tanssi-1000.json",
             "collators": [
                 {
-                    "name": "FullNode-2000",
-                    "command": "../target/release/container-chain-simple-node",
+                    "name": "FullNode-1000",
+                    "command": "../target/release/tanssi-node",
```
Comment (tmpolaczyk) [inline test/configs/zombieStarlight.json:54]: Now I see "dancebox/800" as the chain id in port 9948, this needs to be 
```suggestion
                    "command": "../target/release/container-chain-simple-node",
```

File: test/configs/zombieStarlight.json [hunk 2]
```diff
@@ -46,15 +46,45 @@
     },
     "parachains": [
         {
-            "id": 2000,
+            "id": 1000,
             "chain_spec_path": "specs/single-container-tanssi-1000.json",
```
Comment (tmpolaczyk) [inline test/configs/zombieStarlight.json:50]:
```suggestion
            "chain_spec_path": "specs/single-container-template-container-2000.json",
```

File: test/configs/zombieStarlight.json (hunk 1 above)
Comment (tmpolaczyk) [reply test/configs/zombieStarlight.json:54]: Also this is a full node, those should be defined outside of the `collators` array

Comment (girazoki) [reply test/configs/zombieStarlight.json:54]: no, I would not be able to create blocks if that is the case. Remember our node is not yet programmed to support starlight containbers, that is why I am directly putting dancebox as a container with their own collators

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**